README.md

```diff
@@ -163,6 +165,7 @@
 ### Rust
 
 * [cargo2nix](https://github.com/cargo2nix/cargo2nix) - Granular caching, development shell, Nix & Rust integration.
+* [crane](https://github.com/ipetkov/crane) - A Nix library for building cargo projects.
```

Comment:
Might be good to add in that it provides incremental artifact caching.

Ex: `A Nix library for building Cargo projects with incremental artifact caching.`